packages/jest-jasmine2/src/jasmineAsyncInstall.ts

```diff
@@ -170,6 +174,9 @@ function makeConcurrent(
 
     return originalFn.call(env, specName, () => promise, timeout);
   };
+
+  // @ts-ignore
```

Comment:
I'm not sure about this line.
The return of this function seems to be `ItConcurrentBase` and needs an each function to be present.
We bind the each after calling `jasmineAsyncInstall`, so I added the `ts-ignore`.
Or would an empty each function be better than `ts-ignore`

Reply:
Maybe the each install should happen in here? cc/ @SimenB 

(Not to familiar with this section of the codebase)